Configuration/Routes.yaml

```diff
@@ -15,7 +15,7 @@
 
 -
   name:  'Monocle API'
-  uriPattern: 'api/{@action}'
+  uriPattern: 'monocle/api/{@action}'
```

Comment:
Please remove the `monocle` path prefix, it breaks the routing for existing installations. This applies to all occurrences of `monocle/` in this file.

Reply:
That is problematic since in that case the automatically included route will be preview/... . This is a bit too unspecific and will possibly collide with other pathes. That is why i added the "monocle" prefix. 

BTW: that was why i originally targeted this for 3.0 beacause of the breakiness.